src/e2e/specs/dashboard.spec.ts

```diff
@@ -61,6 +61,34 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Headers`, () =
     expect(await dashboardPage.actionNeededTab.isVisible()).toBeTruthy();
     expect(await dashboardPage.fixedTab.isVisible()).toBeTruthy();
   });
+
+  test("Verify that the Fixed tab layout and tooltips are displayed correctly", async ({
+    dashboardPage,
+  }) => {
+    // link to testrail
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2301532",
+    });
+
+    // verify fixed tab's tooltips and popups
+    await dashboardPage.fixedTab.click();
+    await expect(dashboardPage.heresWhatsFixedCardTitle).toHaveText(
+      "Here’s what we fixed",
+    );
+    await expect(dashboardPage.fixedHeading).toBeVisible();
```

Comment:
Also unrelated, but noticed in the earlier tests (L61) we did something like this:

```js
expect(await dashboardPage.actionNeededTab.isVisible()).toBeTruthy();
```

But here we're using a slightly different (albeit prettier) syntax:
```js
await expect(dashboardPage.fixedHeading).toBeVisible();
```

Reply:
good catch, will be moving away from truthy, will update